tests/e2e/restart_test.go

```diff
@@ -73,7 +74,8 @@ func (s *RestartTestData) SetUpSuite(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// Create a volume on the second datastore
-	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], s.volumeName+"@"+ds2)
+	ds2 = strings.Replace(ds2, "'", "", -1)
+	out, err = dockercli.CreateVolume(s.config.DockerHosts[1], inputparams.CreateFQVN(s.volumeName, ds2))
```

Comment:
config.Datastores already has quotes attached to ds names so adding two back-to-back single quotes fails the command
```
root@sc-rdops-vm02-dhcp-52-237:~# docker volume create -d vsphere --name ''RefcountTest_volume_634502@ shared   Vmfs   - (10)''
-bash: syntax error near unexpected token `('
```
So either we can call CreateFQVN() to add quotes or we can go back to the way it was - s.volumeName+"@"+ds2